atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -506,7 +506,7 @@ def __init__(self, signing_intents, default_signing_intent):
 
         self.signing_intents = []
         # Signing intents are listed in reverse restrictive order in configuration.
-        for restrictiveness, intent in enumerate(reversed(signing_intents)):
+        for restrictiveness, intent in enumerate(reversed(deepcopy(signing_intents))):
```

Comment:
Could we add a comment here explaining why the deepcopy is required at this point?

Reply:
+1